contributors/THIRD-PARTY-EXTENSION.md

```diff
@@ -0,0 +1,104 @@
+## Introduction
+
+Welcome to the guide for developing external extensions in the `create-eth` repository. This document outlines the workflow for creating and testing your extensions.
+
+## Workflow Overview
```

Comment:
Before this: I'd add more context/big picture here too. Starting straight with the "dev" details would be confusing to people (not for use, because we already know the deal).

E.g. of big picture stuff
- Extensions are stored in Github repos
- They follow the template stuff (link to TEMPLATE doc)
- ...